### ISSUE TITLE
Brand colors do not pass WCAG color contrast check

### ISSUE THREAD

**Issue by danrabbit** [mirkobrombin/Bottles#12]
https://github.com/mirkobrombin/Bottles/blob/10740236b52a8a136ac53b35930b3d542d957ae7/data/com.github.mirkobrombin.bottles.appdata.xml#L51

These colors should pass the [WCAG color contrast standard](https://snook.ca/technical/colour_contrast/colour.html#fg=F0E7E7,bg=F0E5F2)

**Comment by mirkobrombin** [comment]
Should be fixed on release [0.0.5](https://github.com/mirkobrombin/Bottles/releases/tag/0.0.5)
